BUTTON_IDEAS.md

```diff
@@ -60,6 +60,7 @@ When adding ideas, please add them in the right category (or add a new one if yo
 | &#9744; | **Dark Mode Toggle**  | A button toggling dark mode.                               | [Harshit Virmani](https://github.com/hvirmani)    |
 | &#9745; | ~~**Disabled Buttons**~~  | A button that has disabled style and non-clickable ability | [Zahra Abdullah](https://github.com/zahra-abdullah) |
 | &#9744; | **Spooky Button** | A button with dramatic/spooky touch to it for special website designs. | [Shristi Singh](https://github.com/confusedcoder1)    |
+| &#9744; | **Wheel button** | A wheel as a button, rotating on hover | [Akshat Garg](https://github.com/akshatgarg12)    |
```

Comment:
Rename it to Loading Button and change description to a button with CSS loading

Reply:
okay.
but I believe this wheel button is also possible in CSS?

Reply:
It is but I think it requires some nested element and currently, we're not going for that. We want the button as simple as possible.

Reply:
okay thanks!